File: RhNetAPI/RhNetAPI/Models/Adm/ApplicationUserModel.cs
using RhNetAPI.Entities.Adm;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RhNetAPI.Models.Adm
{
    public class ApplicationUserModel
    {

        public string UserId { get; set; }
        public string UserName { get; set; }
        public string Email { get; set; }
        public string Cpf { get; set; }
        public List<Client> clients { get; set; }
        public List<ApplicationRole> applicationRoles { get; set; }
        public List<Permission> permissions { get; set; }
    }
}

File: RhNetAPP/src/app/components/models/adm/applicationUser.model.ts
import { UserRole } from 'src/app/components/models/adm/userRole.model';
import { Client } from 'src/app/components/models/adm/client.model';
import { UserPermission } from 'src/app/components/models/adm/userPermission.model';

export interface ApplicationUser {
	userId: string,
	cpf: string,
	userName: string,
	email: string,	
	applicationRoles: UserRole[],
	clients: Client[],
	permissions: UserPermission[]
}